mappings/net/minecraft/block/entity/ChestBlockEntity.mapping

```diff
@@ -1,4 +1,15 @@
 CLASS net/minecraft/class_5279 net/minecraft/block/entity/ChestBlockEntity
+	FIELD field_25093 tickedNeighbours Z
+	FIELD field_25094 northChest Lnet/minecraft/class_5279;
+	FIELD field_25095 eastChest Lnet/minecraft/class_5279;
+	FIELD field_25096 westChest Lnet/minecraft/class_5279;
+	FIELD field_25097 southChest Lnet/minecraft/class_5279;
 	FIELD field_25100 viewerCount I
 	FIELD field_25101 inventory Lnet/minecraft/class_6207;
 	FIELD field_25102 ticksOpen I
+	METHOD method_26953 getChest (Lnet/minecraft/class_6196;)Lnet/minecraft/class_5279;
+		ARG 1 direction
+	METHOD method_26954 initDataFixer (Lnet/minecraft/class_2656;)V
+	METHOD method_26955 isChestBlock (Lnet/minecraft/class_6183;)Z
```

Comment:
`isChest` maybe? Look at upstream, its `AbstractRailBlock.isRail` and not `AbstractRailBlock.isRailBlock`